tests/rptest/tests/wasm_basic_test.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import os
+import uuid
+
+from ducktape.mark.resource import cluster
+from ducktape.utils.util import wait_until
+
+from kafka import KafkaConsumer, TopicPartition
+
+from rptest.clients.rpk import RpkTool
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.clients.kafka_cli_tools import KafkaCliTools
+from rptest.clients.types import TopicSpec
+
+
+class WasmBasicTest(RedpandaTest):
+    topics = (TopicSpec(name="myInputTopic",
+                        partition_count=3,
+                        replication_factor=3,
+                        cleanup_policy=TopicSpec.CLEANUP_DELETE), )
+
+    topic = "myInputTopic"
+    materialized_topic = "myInputTopic.$myOutputTopic$"
```

Comment:
omit the hardcoded `name` from the `TopicSpec` above and a random name will be assigned. Then add a helper method that constructs the materialized topic named using `self.topics[0].name` for the input topic.

Reply:
I can't assign a random topic because our input topic is hardcoded into the fixed script we will launch for this test. Will do the other stuff you mentioned. I'll also leave a note to revert this once we can build our own scripts through our tests on the fly.

Reply:
oh bummer. I thought they were pure functions matched up external. anyway, this is fine then